### DISCUSSION TITLE
Does labevents include the patient's laboratory test records during the ICU?

### DISCUSSION THREAD

**Discussion by huangzhuohua** [MIT-LCP/mimic-code#1807] [MIMIC-IV]
There is a labevents table under the mimiciv_hosp schema. Does its data **include** the test data of the patient during the stay in the ICU? Or are the test data of the patient during the stay in the ICU **only** stored in chartevents under the mimiciv_icu schema?

**Comment by alistairewj** [comment]
Yes, the *labevents* table stores all laboratory measurements for the patient, independent of whether they are in the ICU or not. There are duplicate lab measurements in the *chartevents* table, which you should be aware of. If you are using MIMIC-III, these measurements may be off by 1-hour due to a bug (see #784). Personally I stick to using *labevents* only, most of the time.
